pylzy/tests/test_data/cassettes/env/explorer/test_classify/test_classify_pypi_packages.yaml

```diff
@@ -79,14 +81,2341 @@ interactions:
       X-PyPI-Last-Serial:
       - '17870111'
       X-Served-By:
-      - cache-iad-kcgs7200176-IAD, cache-jnb7020-JNB
+      - cache-iad-kcgs7200176-IAD, cache-cpt13820-CPT
       X-Timer:
-      - S1693839289.416929,VS0,VE237
+      - S1696611834.523621,VS0,VE1
       X-XSS-Protection:
       - 1; mode=block
       content-length:
       - '3755'
     status:
       code: 200
       message: OK
+- request:
```

Comment:
looks too huge to place it in repo

Reply:
I think 2*2500 is not a big deal for git.
We definitely could shrink it, but it is extra work I don't want to do.